gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/store/ManagedDirectBufferAllocator.java

```diff
@@ -79,12 +64,6 @@ public static long defaultMaxMemory() {
         .getTotalPhysicalMemorySize() << 2) / 5 + 7) >>> 3) << 3;
   }
 
-  public ManagedDirectBufferAllocator initialize() {
-    this.logger = initLogger();
-    DirectBufferAllocator.setInstance(this);
-    return this;
-  }
```

Comment:
Ok, I guess you can move the logger initialization to base and remove this. Or tag as override as you mention.